homework/content.js

```diff
@@ -1,37 +1,68 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+/**
+ * @typedef {Object} Note
+ * @property {String} title
+ * @property {String} passages
+ */
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
+(function(exports) {
+
+  /**
+   * Create a instance of ContentManager
+   *
+   * @constructor
+   * @this {ContentManager}
+   */
+  var ContentManager = function () {
+    this._wrapper = document.querySelector('#note-content-wrapper');
+  }
+
+  /**
+   * Add an event handler for note-open event which draw correct note contents
+   *
+   * @this {ContentManager}
+   */
+  ContentManager.prototype.start = function () {
+    window.addEventListener('note-open', (function(event) {
```

Comment:
You should bind the `listener` with `this`, rather than an anonymous function like this.